app/v1/models/domain/StatusReasonEnum.scala

```diff
@@ -29,35 +29,35 @@ object StatusReasonEnum {
     val downstreamValue = "Sign up - return available"
   }
 
-  case object signUpNoReturnAvailable extends StatusReasonEnum {
+  case object `Sign up - no return available` extends StatusReasonEnum {
     val downstreamValue = "Sign up - no return available"
   }
 
-  case object itsaFinalDeclaration extends StatusReasonEnum {
+  case object `ITSA final declaration` extends StatusReasonEnum {
     val downstreamValue = "ITSA final declaration"
   }
 
   case object `ITSA Q4 declaration` extends StatusReasonEnum {
     val downstreamValue = "ITSA Q4 declaration"
   }
 
-  case object cesaSaReturn extends StatusReasonEnum {
+  case object `CESA SA return` extends StatusReasonEnum {
     val downstreamValue = "CESA SA return"
   }
 
-  case object complex extends StatusReasonEnum {
+  case object `Complex` extends StatusReasonEnum {
```

Comment:
this one doesn't need the tick-marks, just the ones with spaces or funny symbols